src/wp/SplashScreen.cs

```diff
@@ -163,5 +190,30 @@ public void hide(string options = null)
                 story.Begin();
             });
         }
+
+        private void StartAutoHideTimer()
+        {
+            var timer = new DispatcherTimer() { Interval = TimeSpan.FromMilliseconds(preferences.SplashScreenDelay) };
+            timer.Tick += (object sender, EventArgs e) =>
```

Comment:
Suggest: Promote the Tick handler to a member function.  In the callback, remove the Tick handler.  This eliminates the circular reference which is created by auto-referencing 'this' in your callback.